submissions/Vladfed92/task_html_css_popup/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./CSS/main.css">
+	<title>HTML_CSS_Popup</title>
+</head>
+
+<body>
+	<header class="header">
+		<nav class="navigation">
+			<ul class="nav__list">
+				<li class="nav__item"><a href="#">Mail</a></li>
+				<li class="nav__item"><a href="#">WTF</a></li>
+				<li class="popup">
+
+					<label for="check" class="check"><img src="./img/popup-button.png" alt=""></label>
+						<input type="checkbox" id="check" name="check">
+						
+
+					<ul class="popup__list">
+						<li class="popup__icon"><a href="#"><img src="./img/news.png" alt="news">News</a></li>
+						<li class="popup__icon"><a href="#"><img src="./img/apps.png" alt="apps">Apps</a></li>
+						<li class="popup__icon"><a href="#"><img src="./img/blogs.png" alt="blogs">Blog</a></li>
+						<li class="popup__icon"><a href="#"><img src="./img/calendar.png" alt="calendar">Calendar</a></li>
+						<li class="popup__icon"><a href="#"><img src="./img/chat.png" alt="chat">Chat</a></li>
+						<li class="popup__icon"><a href="#"><img src="./img/classroom.png" alt="classroom">Classroom</a></li>
+						<li class="popup__icon"><a href="#"><img src="./img/documents.png" alt="documents">Documents</a></li>
+						<li class="popup__icon"><a href="#"><img src="./img/maps.png" alt="maps">Maps</a></li>
+						<li class="details">
+							<details>
+								<summary>More</summary>
```

Comment:
Logically when the second part of the popup is opened this button should be hidden.